### DISCUSSION TITLE
Move the status control into a full "experiment/simulation manager"

### DISCUSSION THREAD

**Discussion by AlexisDrogoul** [gama-platform/gama#100] [Ideas]
Right now, the status control is used to display information about the experiment and the simulations. It is not extremely practical but does the job of keeping a clean and compact UI. 

![An… 2021-09-08 17-06-05](https://user-images.githubusercontent.com/579256/132490195-cbaee42e-5af8-4d69-a3d2-c46d9cc28c28.png)

However, when multiple simulations are run, the limit of this small output become more obvious, especially if one wants to follow the execution of the simulations in parallel or easily jump from one simulation to the other.
As a matter of fact, besides displaying the information, it doesn't play an active role in managing the experiment and the simulations. One could, for example, want to pause or destroy a simulation, etc. or change (like suggested in #2970) the parameters of one specific simulation, or display only this simulation and not the others, etc. 

So I propose that we rely on this drop-down (and probably redesign it, and reorganise a bit the toolbar and parameters view) to accomplish this: 
- selecting the experiment would display the experiment parameters (if any) and all the simulations+permanent outputs 
- selecting one simulation would display only this simulation, its parameters, its views, etc.
- the parameters view would always correspond to the currently displayed experiment/simulation
- at the top of the parameters view would be displayed the time/cycle information of the current exp/sim (and maybe, in the case of experiment, the information about all the simulations + controls over them, like stop/pause/goto...? )
- the controls in the toolbar would apply to the current exp/sim as well. i.e. pause or destroy it. 

Please share your thoughts about this @gama-platform/core !

**Comment by sriramab** [comment]
I would slightly link it with #3058 , especially the ability to select from many.

**Comment by sriramab** [comment]
A general idea - filter. 
I do not mean all the graphs (as you see in the gif), but just a filter ...because I wonder that a dropdown would make the GUI look heavy. 

![image](https://user-images.githubusercontent.com/12668606/132498996-f983ecd5-e708-4ddb-963a-782fb3a2ac4a.gif)

**Comment by AlexisDrogoul** [comment]
@sriramab : sorry, I dont get it. A filter for what ? And what would be the link with the population table ? This enhancement is to simplify the access to the experiment and individual simulations (and clearly separate their displays, parameters, etc. ). So, instead of having a dropdown that does not allow to choose a simulation for instance (except to only display its time/cycle), I propose to turn it into something useful in that department: choosing a simulation would change what is displayed and the parameters.

**Comment by sriramab** [comment]
Yes. Imagine, you have a simulation table similar to population table. In this table, you will have a row name = experiment name/number, and column names are the parameters. Next to the row name, or on hovering over it, one could see a stop, pause for each simulation. there could also be a check box next to each experiment which allows to select experiment that should be shown. 

So if you have columns of parameters of an experiment, then as shown in the gif, one could filter to say.. select all experiments (rows) that satisfy a criteria. for example, all experiments with parameter p = 10.

**Comment by sriramab** [comment]
I linked it to the population table because population table also has the same problem.. ie., too many attributes to scroll to select/toggle individually. So a similar filtering ability can help/simplify also the population table.
